docs/ui/components/Button/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { css } from '@emotion/react';
+import { typography } from '@expo/styleguide';
+import React, { forwardRef, DOMAttributes } from 'react';
+
+type Props = DOMAttributes<HTMLButtonElement | HTMLAnchorElement> & {
+  testID?: string;
+};
+
+export const ButtonBase = forwardRef<HTMLButtonElement, Props>(function ButtonBase(props, ref) {
+  const { testID, ...rest } = props;
```

Comment:
Cleaned almost all props, except `testID`, because they are  defined in `HTMLButtonElement` types and handled by `...rest`